web/react/components/textbox.jsx

```diff
@@ -41,6 +43,7 @@ export default class Textbox extends React.Component {
         this.addedMention = false;
         this.doProcessMentions = false;
         this.mentions = [];
+        this.value = this.props.messageText;
```

Comment:
Why create a new var to hold the prop? Why not just access the prop itself where needed?